src/operator/tensor/matrix_op.cc

```diff
@@ -363,6 +363,8 @@ Examples::
 .set_attr_parser(ParamParser<SliceAxisParam>)
```

Comment:
We should add `MXNET_ADD_SPARSE_OP_ALIAS` to this op. We should also update the description for output storage type (see sparse ops like add_n)